pages/linux/fcrackzip.md

```diff
@@ -0,0 +1,27 @@
+# fcrackzip
+
+> ZIP archive password cracking.
+
+- Bruteforce password with length 4 to 8 with only lowercase, uppercase and digit characters (order matters):
+
+`fcrackzip -b -l 4-8 -c aA1 {{archive}}`
```

Comment:
Are there any long forms of these options? In tldr pages we prefer to use the long argument forms (e.g. `--length` instead of `l`) for clarity. The same goes for the examples below too.

Reply:
Adapted as of your suggestion.
Please note that, in an operational context, I (and I suppose many people) will never use the long form of options. However, you are completely right, it's better for clarity.